qe_metrics/libs/config.py

```diff
@@ -0,0 +1,17 @@
+from pyaml_env import parse_config
+from simple_logger.logger import get_logger
+
+
+class Config:
```

Comment:
why do we need a dedicated class for config? 
there should be separate classes for db and jira.
these classes should inherit from a parent one which will have the configuration attributes and other things that both classes need. this class should have a meaningful name , not `Config`, e.g `QeMetrics`

Reply:
@rnetser I think that makes more sense than my current approach. I will change this today and add some basic classes for the DB and Jira.